src/routes/github/configuration/github-configuration-router.test.ts

```diff
@@ -305,7 +308,7 @@ describe("Github Configuration", () => {
 						jiraHost
 					})
 				)
-				.expect(404);
+				.expect(500);
```

Comment:
500?  None of the tests should ever return a 500, that means something has failed spectacularly...

Reply:
Good catch, have refactored the code so that it handles the error appropriately.